core/src/main/java/com/scalar/db/common/CoreError.java

```diff
@@ -808,6 +808,12 @@ public enum CoreError implements ScalarDbError {
       "Db2 does not support column type conversion from %s to %s",
       "",
       ""),
+  JDBC_TIDB_UNSUPPORTED_COLUMN_TYPE_CONVERSION(
+      Category.USER_ERROR,
+      "0241",
+      "TiDB does not support column type conversion from %s to %s",
```

Comment:
[Future improvement idea] This is not related to this PR at all, but just a memo for future improvements. I think we can probably generalize these kinds of error messages to parameterize the RDBMS name like "%s doesn't support ...." so that we don't need to add new error messages for each database.